setup/create-wp-project/src/replace-version-numbers.js

```diff
@@ -0,0 +1,73 @@
+
+const replace = require('replace-in-file');
+const path = require('path');
+const fs = require('fs-extra');
+
+const replaceVersionNumbers = async (projectPath, themeName) => {
+	const styleCssPath = path.join(projectPath, 'style.css');
+	const functionsPhpPath = path.join(projectPath, 'functions.php');
+	const packageLockPath = path.join(projectPath, 'package-lock.json');
+	const packageJsonPath = path.join(projectPath, 'package.json');
+	const changelogPath = path.join(projectPath, 'CHANGELOG.md');
+
+	// style.css
+	if (await fs.pathExists(styleCssPath)) {
+		await replace({
+			files: styleCssPath,
+			from: /^Version: .*$/m,
```

Comment:
Should the empty spaces be referenced with `\s` in regex?

Reply:
Tried it, works as is 🤷 
Internet says ` ` and `\s` will do the same 😄

Reply:
I mean, the `\s` will match any whitespace character (from regex101: \s matches any whitespace character (equivalent to [\r\n\t\f\v ])), so I'm not sure if that could have some unintended consequences or not 😅